internal/multicluster/internal/types/types.go

```diff
@@ -18,4 +18,5 @@ func Register(r resource.Registry) {
 	RegisterNamespaceExportedServices(r)
 	RegisterPartitionExportedServices(r)
 	RegisterComputedExportedServices(r)
+	RegisterEnterpriseTypes(r)
```

Comment:
NIT: Might be better to have a line break before this or have this call as the first invocation in the function.